app/gui/src/dashboard/assets/inbox_with_notification.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M5 9H2V13C2 13.5523 2.44772 14 3 14H13C13.5523 14 14 13.5523 14 13V9H11C10.4477 9 10 9.44772 10 10C10 11.1046 9.10457 12 8 12C6.89543 12 6 11.1046 6 10C6 9.44772 5.55228 9 5 9Z" fill="black" stroke="black"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M10.5351 0C10.1948 0.588347 10 1.27143 10 2H3C2.44772 2 2 2.44772 2 3V13C2 13.5523 2.44772 14 3 14H13C13.5523 14 14 13.5523 14 13V6C14.7286 6 15.4117 5.80521 16 5.46487V13C16 14.6569 14.6569 16 13 16H3C1.34315 16 0 14.6569 0 13V3C0 1.34315 1.34315 0 3 0H10.5351Z" fill="black"/>
+<circle cx="14" cy="2" r="2" fill="#F3180A"/>
```

Comment:
The red dot should be a component, similarly to: https://chakra-ui.com/docs/components/avatar#badge

Reply:
added as `StatusBadge`